src/rpc_api/mod.rs

```diff
@@ -58,7 +58,7 @@ pub static ACCESS_MAP: Lazy<HashMap<&str, Access>> = Lazy::new(|| {
     // Message Pool API
     access.insert(mpool_api::MPOOL_GET_NONCE, Access::Read);
     access.insert(mpool_api::MPOOL_PENDING, Access::Read);
-    access.insert(mpool_api::MPOOL_PUSH, Access::Write);
+    access.insert(mpool_api::MPOOL_PUSH, Access::Read);
```

Comment:
Anyone can push to mpool? Is it safe?

Reply:
I'm somewhat convinced that it is safe. Lotus has three ways of receiving signed messages: `MPoolPush`, `MPoolPushUntrusted`, and p2p pubsub. `MPoolPushUntrusted` has a few more sanity checks and a lower cap on outstanding messages. But any peer can send messages through p2p pubsub which does not have a limit and acts just like `MPoolPush`. Any security we add to `MPoolPush` is merely security through obscurity since we (and Lotus) blindly accept messages through pubsub.

Reply:
Makes sense, thanks for clarification.

Reply:
Oh, and glif.io doesn't even expose `MPoolPushUntrusted`. But they do let everyone use `MPoolPush`. It's kinda strange. All in all, I think we're best off making `MPoolPush` public and then making it as difficult to spam as possible.

Reply:
Once we have rate-limiting implemented, we could by default limit it to zero.

Reply:
Added a note in the code for why we differ from Lotus.